src/main/java/com/nanalysis/jcamp/parser/ASDFParser.java

```diff
@@ -37,10 +37,14 @@
         SQZ, DIF, DUP, NUMERICAL
     }
 
+    enum Mode {
+        SQZ, DIF
+    }
```

Comment:
Ok, it contains negative and positive digits

Reply:
That's not quite that.

When using DIF (differential mode), the token indicates a difference with the previous value.
When using SQZ (squeezed mode), the token indicates an absolute value.

When a line ends in DIF mode, the next one is supposed to start with a SQZ value repeating the last one, to check consistency and limit rounding errors. 

Then, we also have some tokens indicating DUP (duplicate), where the token tells us how many times to repeat the value. Before this change, when a line ended with a DUP token, we didn't considered the first one of the next line as a consistency check. However, if it ends with a DUP that followed a DIF (meaning: we repeated difference changes, not absolute values), some implementations actually have a Y check on the next line. 

This is what the test does and the reason for the change.